app/src/main/java/com/mghelas/internship_playground/MainActivity.java

```diff
@@ -23,4 +31,18 @@ public void onClick(View v) {
             }
         });
     }
+
+    public void forceCrash(View view) {
```

Comment:
2 questions, why do we have here view as a param, and where we use this method?

Reply:
It is a test snippet from crashlytics, so I copied it as shown in documentation.

I invoke this method on button click defined in main_acticity.xml with android:onClick attribute

Reply:
removed forceCrash method.